core/ui/compose/designsystem/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="core_ui_designsystem_accessibility_hide_password">Hide password</string>
+    <string name="core_ui_designsystem_accessibility_show_password">Show password</string>
```

Comment:
Please add comments for translators so they get a rough idea of where these strings can appear in the app.

Reply:
What about `designsystem_atom_password_textfield_hide_password` and `designsystem_atom_password_textfield_show_password`?